app.json

```diff
@@ -45,6 +45,10 @@
     "ELASTICSEARCH_INDEX": {
       "description": "Index to use on Elasticsearch"
     },
+    "ELASTICSEARCH_INDEXING_CHUNK_SIZE": {
+      "description": "Chunk size to to use for Elasticsearch indexing tasks",
+      "required": false,
```

Comment:
Trailing comma here is invalid JSON.

Reply:
Got rid of the trailing comma (and extra `to`)